sceptre/cli/template.py

```diff
@@ -137,3 +138,33 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="detect-stack-drift", short_help="Detects stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def detect_stack_drift_command(ctx, path):
+    """
+    Detect stack drift on running stacks.
+    \f
+
+    :param path: The path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        output_format=ctx.obj.get("output_format"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.detect_stack_drift()
+    output = "\n".join([
+        json.dumps({stack_name: response}, sort_keys=True, indent=2, default=str)
+        for stack_name, response in responses.values()
+    ])
+    write(output, context.output_format)
```

Comment:
This method should definitely exit with a relevant message and a nonzero exit code in the case where the drift detection failed or timed out.